### DISCUSSION TITLE
我正在尝试将hexo与vitewpress相结合

### DISCUSSION THREAD

**Discussion by everfu** [hexojs/hexo#5536] [Show and tell]
https://github.com/everfu/solitude.js.org
请看仓库，我想通过脚本生成两种语言的静态站，但生成zh时会报错，所以有大佬能帮我看看交个pr或者告诉我解决方案嘛

**Comment by stevenjoezhang** [comment]
能发一下你的脚本的代码吗？是在执行什么命令的时候报错？
